### ISSUE TITLE
Even more random quest suggestions

### ISSUE THREAD

**Issue by chochem** [GTNewHorizons/GT-New-Horizons-Modpack#5457]
#### Which modpack version are you using?
2.0.8.3-8.8
#

I assembled another list of quest suggestions, mostly HV-IV, to add more polishing. 

- [x] Quest 1504 (Divine Rod) Explain how to get magnetic iron rods. This is currently only explained at the end of Tier 1 for electric motors, but divine rod is a tier 0.5 quest. This is a very common question of new players as many naturally look at the magnetic ingot in NEI which does not lead anywhere at this point.  Either move the explanation from motors to the divine rod or have them in both places.

- [x] Quest 1273 (LV Brewery) The text claims regarding brewing stands, that *you can still find them in chests*. As far as I know, that is not the case but I might be wrong who knows. However, you CAN find them standing around in the roguelike dungeon secret room. See my recently posted pictures on discord. Its on the second level hidden behind a wall. There is always a brewing stand inside, but the room does not spawn in every dungeon (rng) and possible not in every dungeon type (not sure on that one).  _Edit: Someone told me they are also available in the Ur-Ghast-tower. Have not checked that though._

- [x] Quest 1012 (Rutile Dust) should not require a HV thermal centrifuge as it is useless for rutile dust. (rutile is secondary, not tertiary) However one could consider a separate quest for the thermal centrifuge themed around gallium from bauxite. (which is tertiary)

- [ ] <s>Add an optional quest for the cinnabar vein on the moon to introduce bartworks ores. Mention that they are separate in NEI.</s>

- [x] Quest 805 (lapotron dust): energium and lapis amounts must be swapped to reflect the correct ratio.

- [x] Remove Quest 1461 (desh), it is really a duplicate of 1891 (desh,scheelite, tungstate)

- [x] Quest 1224: molten epoxid and epoxid sheets should both require 27, not just 4. (they are based on 144, not 1000). Best double check the number.

- [x] Quest 1702 (Potassium Dichromate) has a wrong ratio. It is  (2,2) -> 1 (not (1,1)->1). Just reduce to 4 Potassium Dichromate Dusts (works with quests before and after)

- [x] Quest 2309 (Another Acid to worry about, formic acid) has a wrong ratio, too. Reduce sulfuric acid from 16 to 8.

- [x] Quest 1692 (Nitration mixture) also has a wrong ratio. 10 sulfuric + 10 nitric acid actually give 20 nitration mixture, not just 10.

- [x] Quest 1652 (Power of a jet engine) should be removed. It is just a less fleshed out duplicate of Quest 1894 (Gas Turbine).

- [x] Quest 290 Ninja style. Fix the quest reward. Currently one can get a normal book with enchants on it instead of an enchanted book which is actually usable.

- [x] Quest 1492 (Assembling Line) The number of requested Solid Steel Machine Casings by the quest (39) has no relation with the multiblock that I can recognize. It should be 3x11 - 4 (fluid inputs) -2 (energy) - 1 (maint.) = 26.

- [x] Quest 1493 (Tier 7). It says 'you will need Chrome for machines'. Maybe this was the case at some point, but the main material now is Rhodium-Plated Palladium. The text should be changed accordingly.

- [x] Quest 1715 (On your way to Tier 4 DIMs). First task is to reach the asteroids. That should be removed. This is a T3 dim and has already been done previously.

- [x] All the Quests in the Tier 7 Tab: Replace Chrome Ingots in the Quest Rewards for something else, now that Chrome is no longer the tier-material.

- [x] Quest 1532 (Tier 4 Heavy Duty Plates) This quest is a bit of a mess. The text is telling you to craft 12 Tier 4 Heavy Duty Plates, the ingredients from the first task only suffice for 11, but what one actually needs for the rocket is 13.
so 1) the text should tell you to do 13. 2) second and third task should be adjusted to 13 3) the first task should ask for 14 Heavy duty plates and 78 Compressed Ice Plates, and one should probably add the 52 ruridit bolts as well. 

- [x] The rest of the space race tab: All heavy duty plate quests should demand one more plate from the player (for the schematic!) and the schematic quest should always come after that quest. E.g. Quest 1428 (Tier 2 Schematic) has to be after 1427 (Tier 2 Heavy duty plates) .

**Comment by yuri70072** [comment]
- [ ] Quest 1154 (Magnalium Crossbow Bolts) should note that, at LV, it needs the tinker's variation of molten iron/bronze to make the Magnalium crossbow bolt. The NEI recipe with molten tinker's iron is at page 43 and people generally look for the molten materials made in the fluid extractor,which requires MV.

**Comment by Prometheus0000** [comment]
For Quest 1012 (Rutile Dust) you get rutile from TCing the purified bauxite? And gallium from macerating the result?

**Comment by chochem** [comment]
not sure what you are saying. Let me explain in detail: rutile is a secondary byproduct for both ilmenite and bauxite. a thermal centrifuge is 100% pointless for primary or secondary byproducts. (at best same output as normal centrifuge at way higher energy cost) A thermal centrifuge is only useful for tertiary byproducts, so stuff like gallium. The quest description makes the same mistake.

**Comment by Prometheus0000** [comment]
I forgot you could use the normal centrifuge instead, haha.

**Comment by Prometheus0000** [comment]
About the cinnabar vein on the moon, according to bart, it's supposed to be on mercury. https://discordapp.com/channels/181078474394566657/234936569360809996/693194373831196783
Hopefully he'll fix his ores soon.

**Comment by Prometheus0000** [comment]
I've gone through all of these, but in the future please post them to separate issues, if only some of them are complete, it can't be marked as finished, and who knows which ones are done in that case? Also, you need to specify the tab for each one, or group them accordingly too. Having to hunt them down is very annoying.

**Comment by chochem** [comment]
wow, thanks for all the work!

I did find cinnabar veins on the moon (in 2.0.8.3), but maybe it has been changed since. Anyway, you are probably right that we should wait until bart fixes them fully including the NEI-part and come back to it then.

Sorry about the layout. That was just copy pasted from a previous issue by Bluebine. :P
Will make separate issues and mention the quest tab in the future. :)
